### ISSUE TITLE
Error: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by AMerkuri** [samrith-s/parcel-plugin-structurize#2]
I have a following build script:
`"build": "parcel build index.html"`
Added following to `package.json`

```json
  "parcel-plugin-structurize": {
    "scripts": {
      "match": "*.{js,js.map}",
      "folder": "js"
    },
    "styles": {
      "match": "*.{css,css.map}",
      "folder": "css"
    },
    "assets": {
      "match": "*.{png,svg,jpg,jpg2,jpeg,gif,bmp,webm}",
      "folder": "assets"
    }
  }
```

Get following error on build
```
(node:11712) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open 'C:\Users\User\Downloads\vue\vue\dist\C:\Users\User\Downloads\vue\vue\index.html'
    at Object.openSync (fs.js:451:3)
    at Object.readFileSync (fs.js:351:35)
    at markupFiles.map.file (C:\Users\User\Downloads\vue\vue\node_modules\parcel-plugin-structurize\src\index.js:41:38)
    at Array.map (<anonymous>)
    at Bundler.Structurize.bundler.on (C:\Users\User\Downloads\vue\vue\node_modules\parcel-plugin-structurize\src\index.js:39:45)
    at Bundler.emit (events.js:182:13)
    at Bundler.bundle (C:\Users\User\AppData\Roaming\npm\node_modules\parcel-bundler\src\Bundler.js:361:12)
    at process.internalTickCallback (internal/process/next_tick.js:77:7)
(node:11712) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:11712) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by AMerkuri** [comment]
It works if you replace `Path.join()` with `Path.resolve()`
![image](https://user-images.githubusercontent.com/969989/57861457-52247e80-77ff-11e9-8752-873937380d0c.png)

**Comment by samrith-s** [comment]
Hey @AMerkuri, sorry I couldn't reply earlier. 

I've released a minor version [v1.1.0](https://github.com/samrith-s/parcel-plugin-structurize/releases/tag/v1.1.0). Can you tell me if it works for you?

**Comment by samrith-s** [comment]
Closing due to no response.
